src/components/signupForm/index.tsx

```diff
@@ -24,18 +25,23 @@ const Footer: typeof Paragraph = styled(Paragraph)<ParagraphProps>`
 `;
 
 interface SignupFormProps {
+  readonly formInstance: FormInstance;
   readonly onFinish: (values: SignupRequest) => void;
+  readonly windowType: WindowTypes;
 }
 
-const SignupForm: React.FC<SignupFormProps> = ({ onFinish }) => {
-  const { windowType } = useWindowDimensions();
-
+const SignupForm: React.FC<SignupFormProps> = ({
+  formInstance,
+  onFinish,
+  windowType,
+}) => {
   return (
     <>
-      <Form name="basic" initialValues={{ remember: true }} onFinish={onFinish}>
+      <Form name="basic" form={formInstance} onFinish={onFinish}>
         <FullWidthSpace direction="vertical" size={1}>
           <FormRow>
             <FormHalfItem
+              name="firstName"
               rules={[
```

Comment:
💭  Maybe create a form rules file with all of the rules we use throughout the app.

Reply:
Yeah, I thought about it but the only rules in common between the login/signup form is the one for email. The signup form has more in common with the settings form, so I was just going to pull out the common ones in the ticket to fix settings. You're right though, I'll create a `formRules` file as part of this and update settings later. Do you think we should put all of the rules in there or just the ones used in multiple files?